index.es.md

```diff
@@ -0,0 +1,104 @@
+  <p><strong>Editora:</strong> <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
+  <p>Desarrollado con la colaboración del Grupo de Trabajo de Educación y Divulgación (<a href="http://www.w3.org/WAI/EO/">EOWG</a>).</p>
+ref: /test-evaluate/
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Resumen" class="full" %}
+{:/}
+
+Esta página contiene enlaces a recursos para ayudar a evaluar la accesibilidad web. La evaluación de la accesibilidad también se denomina "análisis", "auditoría" o "prueba".
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Contenidos de la página" class="full" %}
+{:/}
+
+-   TOC is created automatically.
+{:toc}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## Introducción {#intro}
+
+Al desarrollar o rediseñar un sitio web o una aplicación web, evalúe la accesibilidad desde el principio y durante todo el proceso de desarrollo para identificar los problemas de accesibilidad desde el principio, cuando es más fácil resolverlos.
+
+Existen herramientas que ayudan con la evaluación. Sin embargo, ninguna herramienta por sí sola puede determinar si un sitio cumple con los estándares de accesibilidad. Se requieren buenos conocimientos para determinar si un sitio es accesible.
+
+## Pruebas iniciales {#initial}
+
+Incluso si usted no tiene conocimiento sobre accesibilidad y no posee herramientas para evaluarla, aun puede comprobar algunos de los aspectos de la accesibilidad.
+
+[Comprobaciones sencillas — Una primera revisión a la accesibilidad web](/test-evaluate/preliminary/)
+:   Facilita una guía paso a paso, junto con sencillas explicaciones, para evaluar algunos aspectos de la accesibilidad.
+
+## Herramientas
+
+Las herramientas de evaluación de la accesibilidad web son programas de software o servicios en línea que ayudan a determinar si el contenido web cumple con los estándares de accesibilidad.
+
+[Lista de herramientas de evaluación de la accesibilidad web](https://www.w3.org/WAI/ER/tools/)
+:   Incluye información sobre más de 100 herramientas. Puede utilizar los filtros para limitar la lista a los tipos de herramientas que le interesan.
+
+[[Elegir herramientas de evaluación de accesibilidad web]](/test-evaluate/tools/selecting/)
+:   Proporciona orientación sobre la elección de las herramientas. Describe las características y la funcionalidad de los diferentes tipos de herramientas de evaluación y plantea las cosas a tener en cuenta para su situación.
+
+## Evaluación de la conformidad {#conformance}
+
+La evaluación de conformidad determina el grado en que las páginas o las aplicaciones web cumplen los estándares de accesibilidad. La metodología de evaluación de la conformidad de la accesibilidad de sitios web del W3C (WCAG-EM) es un acercamiento para determinar la conformidad con las Pautas de Accesibilidad al Contenido Web (WCAG).
```

Comment:
As the WCAG-EM is a document (a Working Group Note), I'd propose to quote it and start it with a capital letter. Thus, I'd replace 'La metodología de evaluación de la conformidad de la accesibilidad de sitios web del W3C (WCAG-EM)' with 'La "Metodología de evaluación de la conformidad de la accesibilidad de sitios web" (WCAG-EM) del W3C'.